source/as-extendable.ts

```diff
@@ -0,0 +1,18 @@
+/**
+A passthrough type that allows some types to be used to extend an interface, which originally can't.
+
+Use-cases:
+- A member of an interface (which is an object): `JSX.IntrinsicElements["a"]`
```

Comment:
```suggestion
- A member of an interface (which is an object): `JSX.IntrinsicElements['a']`
```

Applies elsewhere too.